tests/saitests/sai_qos_tests.py

```diff
@@ -1051,13 +1174,16 @@ def runTest(self):
                 assert pg_dropped_cntrs[dscp] > pg_dropped_cntrs_old[dscp]
 
         finally:
+            show_stats(self.__class__.__name__, self, self.test_params.get('sonic_asic_type', None), self.test_params.get('test_port_ids', None), bases=stats)
             sai_thrift_port_tx_enable(self.client, asic_type, [dst_port_id])
 
 
 class LosslessVoq(sai_base_test.ThriftInterfaceDataPlane):
     def runTest(self):
         time.sleep(5)
         switch_init(self.client)
+        time.sleep(5)
+        stats = show_stats('just collect base data', self, self.test_params.get('sonic_asic_type', None), self.test_params.get('test_port_ids', None), silent=True)
```

Comment:
there are many repeated code snippet like this, could we put them into a func and call the func in each class? If there is any change later, we don't need to modify the code everywhere. @XuChen-MSFT